libc/utils/docgen/netinet/in.yaml

```
@@ -0,0 +1,57 @@
+macros:
+  IPPROTO_IP:
+    in-latest-posix: ''
+  IPPROTO_IPV6:
+    in-latest-posix: ''
+  IPPROTO_ICMP:
+    in-latest-posix: ''
+  IPPROTO_RAW:
+    in-latest-posix: ''
+  IPPROTO_TCP:
+    in-latest-posix: ''
+  IPPROTO_UDP:
+    in-latest-posix: ''
+  INADDR_ANY:
+    in-latest-posix: ''
+  INADDR_BROADCAST:
+    in-latest-posix: ''
+  INET_ADDRSTRLEN:
+    in-latest-posix: ''
```

The ipv6 variant is missing: `INET6_ADDRSTRLEN`.

```suggestion
  INET_ADDRSTRLEN:
    in-latest-posix: ''
  INET6_ADDRSTRLEN:
    in-latest-posix: ''
```